clusterloader2/pkg/config/cluster.go

```diff
@@ -88,6 +88,7 @@ type PrometheusConfig struct {
 	ScrapeMetricsServerMetrics bool
 	ScrapeNodeLocalDNS         bool
 	ScrapeAnet                 bool
+	ScrapeNetPol               bool
```

Comment:
nit: ScrapeNetworkPolicies [let's avoid shortcuts, especially in configs]

Reply:
done